pom.xml

```diff
@@ -132,6 +132,7 @@
 						</goals>
 						<configuration>
 							<inputSpec>https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-scale-api-definitions/173a41212f49ef95b9c75012eed8247a72053073/cat/CaT-service.yaml</inputSpec>
+							<!-- <inputSpec>https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/OCDS/OCDS_Requirements_Ext.yaml</inputSpec> -->
```

Comment:
I guess this yaml change is intentional (until merged into main on the API repo)?

Reply:
I'll remove it in this PR - was just me trying the generator on the various referenced YAMLs to try and track down issues...